with-react-three-fiber/package.json

```diff
@@ -1,14 +1,12 @@
 {
   "dependencies": {
     "expo": "^43.0.0",
-    "expo-gl": "~11.0.3",
-    "expo-three": "~5.7.0",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
+    "react": "18.0.0-rc.0",
+    "react-dom": "18.0.0-rc.0",
     "react-native": "0.64.3",
     "react-native-web": "0.17.1",
-    "react-three-fiber": "^5.3.19",
-    "three": "^0.122.0"
+    "react-three-fiber": "^8.0.0-beta-04",
```

Comment:
I'm hesitant to merge an update from stable to beta release.

Reply:
I think we can revisit once v8 is stable. Also, three r134+ moved loaders to the fetch API so that's broken for the time being. Should have enough time to resolve that.